src/main/java/de/tum/in/www1/artemis/service/exam/ExamService.java

```diff
@@ -1227,10 +1227,8 @@ public SearchResultPageDTO<Exam> getAllOnPageWithSize(final PageableSearchDTO<St
      * @return exam page
      */
     public Page<Exam> getAllActiveExams(Pageable pageable, final User user) {
```

Comment:
Why is User final, but pageable is not? Please try to make this consistent on method you alter.

Reply:
There is no reason for that. I will change pagable to `final` :+1: